src/app/pages/ConsensusDashboardPage/SnapshotValidators.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { TFunction } from 'i18next'
+import { useGetConsensusValidators, Validator } from '../../../oasis-nexus/api'
+import { SearchScope } from '../../../types/searchScope'
+import { SnapshotCard } from '../../components/Snapshots/SnapshotCard'
+import { PieChart } from '../../components/charts/PieChart'
+import { API_MAX_TOTAL_COUNT } from '../../config'
+
+type ValidatorsStats = {
+  label: string
+  value: number
+}
+
+// TODO: remove if backend will provide validators stats
+function countValidatorsState(t: TFunction, validators: Validator[] | undefined): ValidatorsStats[] {
+  let activeCount = 0
+  let inactiveCount = 0
+  if (!validators) return []
+
+  validators.forEach(validator => {
+    if (validator.active) {
+      activeCount++
+    } else {
+      inactiveCount++
+    }
+  })
+
+  return [
+    { label: t('validator.active'), value: activeCount },
+    { label: t('validator.inactive'), value: inactiveCount },
+  ]
+}
+
+export const SnapshotValidators: FC<{ scope: SearchScope }> = ({ scope }) => {
+  const { t } = useTranslation()
+  const { network } = scope
+
+  const validatorsQuery = useGetConsensusValidators(
+    network,
+    { limit: API_MAX_TOTAL_COUNT },
+    {
+      query: {
+        cacheTime: 0,
+      },
+    },
+  )
+  const validators = validatorsQuery.data?.data.validators
+
+  return (
+    <SnapshotCard title={t('validator.listTitle')}>
+      {!!validators?.length && (
```

Comment:
Is this a replacement for loading state, or a guard for bad API?

Reply:
guard, but there is another one in `countValidatorsState` so I can kill this one